package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "geek-notes",
+  "version": "1.0.0",
+  "description": "Notes Web Application",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FACK1/Geek-Notes.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FACK1/Geek-Notes/issues"
+  },
+  "homepage": "https://github.com/FACK1/Geek-Notes#readme",
+  "devDependencies": {
+    "eslint": "^5.10.0",
+    "nodemon": "^1.18.8",
+    "supertest": "^3.3.0",
+    "tap-spec": "^5.0.0",
+    "tape": "^4.9.1"
+  },
+  "dependencies": {
+    "cookie": "^0.3.1",
+    "env2": "^2.2.2",
+    "jsonwebtoken": "^8.4.0",
```

Comment:
I would say, you shouldn't add dependencies before you actually use them. 

However I'm happy to merge with them still here.